packages/grid/data-grid/src/tests/keyboard.DataGrid.test.tsx

```diff
@@ -127,13 +120,13 @@ describe('<DataGrid /> - Keyboard', () => {
   });
 
   const KeyboardTest = (props: { nbRows?: number }) => {
-    const data = useData(props.nbRows || 100, 20);
```

Comment:
why change that? I believe it's better to test with virtualization

Reply:
I assumed 20 was enough to trigger virtualization, 360px height. But agree, this change is unrelated to what I'm going after, better isolate it.

Reply:
Actually there are no tests that test virtualization here (unless I'm missing something)

Reply:
Well it's an integration test, as we had 100 rows we test that it scroll to the bottom and so on with the different keys. And for it to work properly, the virtualization needs to work

Reply:
ok, updating it